benchmark/spmv/spmv.cpp

```diff
@@ -76,7 +76,9 @@ void validate_option_object(const rapidjson::Value &value)
 // Command-line arguments
 DEFINE_string(formats, "coo",
               "A comma-separated list of formats to run."
-              "Supported values are: coo, csr, ell, sellp, hybrid");
+              "Supported values are: coo, csr, ell, sellp, hybrid, hybrid0, "
+              "hybrid25, hybrid33, hybridlimit0, hybridlimit25, hybridlimit33, "
+              "hybridminstorage");
```

Comment:
Please add here a description what the different formats are:
" coo: Coordinate storage. The CUDA kernel uses the load-balancing approach suggested in Flegar et al.: Overcoming Load Imbalance for Irregular Sparse Matrices."
"csr: Compressed Sparse Row storage. The CUDA kernel invokes NVIDIAs cuSPARSE CSR routine."
"ell: Ellpack format according to Bell and Garland: Efficient Sparse Matrix-Vector Multiplication on CUDA."
"sellp: Sliced Ellpack using a default block size of 32"
...

Reply:
done